articles/virtual-machines/trusted-launch-existing-vm.md

```diff
@@ -297,7 +297,7 @@ Follow the steps to enable Trusted Launch on an existing Azure Generation 2 VM b
 
 ## Azure Advisor Recommendation
 
-Azure Advisor populates an **Enable Trusted Launch Compatible Images and VM Sizes for Existing Generation 2 VMs** operational excellence recommendation for existing Generation 2 VMs to adopt [Trusted Launch](trusted-launch.md), a higher security posture for Azure VMs at no additional cost to you. Ensure Generation 2 VM has all prerequisites to migrate to Trusted Launch, follow all the best practices including validation of OS image, VM Size, and creating restore points. For the Advisor recommendation to be considered complete, follow the steps outlined in the [**Enable Trusted Launch on an existing VM**](trusted-launch-existing-vm.md) to upgrade the virtual machines security type and enable Trusted Launch. 
+Azure Advisor populates an **Enable Trusted Launch foundational excellence, and modern security for Existing Generation 2 VM(s)** operational excellence recommendation for existing Generation 2 VMs to adopt [Trusted Launch](trusted-launch.md), a higher security posture for Azure VMs at no additional cost to you. Ensure Generation 2 VM has all prerequisites to migrate to Trusted Launch, follow all the best practices including validation of OS image, VM Size, and creating restore points. For the Advisor recommendation to be considered complete, follow the steps outlined in the [**Enable Trusted Launch on an existing VM**](trusted-launch-existing-vm.md) to upgrade the virtual machines security type and enable Trusted Launch. 
```

Comment:
@Howie425 - I already made this change, request to pull in current version.
Only change pending is the above VMs VMs line. You can pull in current version and re-publish PR.

Ensure `Trusted launch` camel-case. L should be small-case in Launch.